### DISCUSSION TITLE
Sublime Text Key Mapping with Neovintageous

### DISCUSSION THREAD

**Discussion by MichaelMFox** [gerardroche/gerardroche.github.io#53] [General]
Is this the right way to obtain `ctrl-[` for escape (to command mode) for Neovintageous ?

```
{
  "keys": ["ctrl+["], 
  "command": "_enter_normal_mode", 
  "context":
            [
                { "key": "setting.command_mode", "operand": false },
                { "key": "setting.is_widget", "operand": false }
            ]
},
```

**Comment by gerardroche** [comment]
Try

```
    {
        "keys": ["ctrl+["],
        "command": "_nv_feed_key",
        "args": {"key": "<esc>"},
        "context": [{"key": "vi_insert_mode_aware"}]
    },
```

Note that ctrl keys are disabled by default. When ctrl keys are enabled you don't need to create this custom mapping because it will be enabled by default.

**Comment by MichaelMFox** [comment]
Thank you Gerard.  I simply cannot get it.  I am back into Sublime, having
purchased it,
so not an expert. But for this issue I had to use AutoHotKey to make Ctrl-[
work as Esc key with Neovintageous.
I could only get Ctrl-[ to work in Vintage mode.  But of course I needed
Neo.

**Comment by gerardroche** [comment]
Try adding:

```
    {"keys": ["ctrl+["], "command": "nv_feed_key", "args": {"key": "<C-[>"}, "context": [{"key": "nv_command_or_insert"}, {"key": "nv_handle_key", "operand": "<C-[>"}]},

```
